CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This is the log of notable changes to EAS CLI and related packages.
 ### 🛠 Breaking changes
 
 ### 🎉 New features
+- Added flag `--emit-build-meta` to emit `eas-update-metadata.json` in the bundle folder with detailed informations about the generated bundle(s)
```

Comment:
```suggestion

- Added flag `--emit-build-meta` to emit `eas-update-metadata.json` in the bundle folder with detailed informations about the generated bundle(s). ([#2451](https://github.com/expo/eas-cli/pull/2451) by [@rainst](https://github.com/rainst))
```

Reply:
I have updated this entry in the changelog (I replaced also with the new flag)